src/Microsoft.Fx.Portability/ApiPortService.cs

```diff
@@ -156,29 +157,40 @@ public ApiPortService(string endpoint, ProductInformation info)
             return await _client.CallAsync<IEnumerable<ResultFormatInformation>>(HttpMethod.Get, Endpoints.ResultFormat);
         }
 
+        public async Task<ServiceResponse<ResultFormatInformation>> GetDefaultResultFormatAsync()
+        {
+            return await _client.CallAsync<ResultFormatInformation>(HttpMethod.Get, Endpoints.DefaultResultFormat);
+        }
+
         public void Dispose()
         {
             _client.Dispose();
         }
 
         private async Task<IEnumerable<ResultFormatInformation>> GetResultFormat(IEnumerable<string> format)
```

Comment:
I know this was there before, but to make it consistent, can we rename this to `GetResultFormatsAsync(IEnumerable<string> formats)` since this is returning multiple formats and it is an asynchronous call.